### PR TITLE
Update terms.json

### PR DESCRIPTION
**Pull request by FrancescoXX**
updated terms relative to the blockchain. Hacktoberfest contribution

### CODE REVIEW

File: data/terms.json [hunk 1]
```diff
@@ -582,5 +582,37 @@
   {
     "term": "Fungible",
     "meaning": "Fungible is a term used to describe a token of same kind that can be transferred from one address to another."
+  },
+  {
+    "term": "ABI",
+    "meaning": "The standard way to interact with contracts in the Ethereum ecosystem, both from outside the blockchain and for contract-to-contract interaction."
```
Comment (Olanetsoft) [inline data/terms.json:588]: Some minor update here.

Kindly make it `ABI is the standard ...` not `The standard ...`

File: data/terms.json [hunk 2]
```diff
@@ -582,5 +582,37 @@
...
+  },
+  {
+    "term": "Byzantine Fault Tolerance",
+    "meaning": "Byzantine fault is a condition where components of a distributed computer system may fail, with imperfect information on whether a component has failed."
```
Comment (Olanetsoft) [inline data/terms.json:592]: This is okay ✅

File: data/terms.json [hunk 3]
```diff
@@ -582,5 +582,37 @@
...
+  },
+  {
+    "term": "finality",
+    "meaning": "It's the assurance that blockchain transactions, once completed, cannot be altered, reversed canceled. The blockchain latency level affects the finality rate."
```
Comment (Olanetsoft) [inline data/terms.json:596]: Some minor update here.

Kindly make it `Finality is the ...` not `It's the assurance ...`

File: data/terms.json [hunk 4]
```diff
@@ -582,5 +582,37 @@
...
+  },
+  {
+    "term": "IPFS",
+    "meaning": "It stands for InterPlanetary File System. It's a protocol, an hypermedia, a file sharing p2p network. It's used for storing/sharing data in a distributed file system. It uses content-addressing to uniquely identify each file in a global namespace connecting IPFS hosts."
```
Comment (Olanetsoft) [inline data/terms.json:600]: Some minor update here.

Kindly make it `IPFS stands for ...` not `It stands for ...`

File: data/terms.json [hunk 5]
```diff
@@ -582,5 +582,37 @@
...
+  },
+  {
+    "term": "Merkle tree",
+    "meaning": "A Merkle tree (also called hash tree) is a tree in which leaf nodes are labeled with the cryptographic hash of a data block and every non-leaf node is labeled with the cryptographic hash of the labels of its child."
```
Comment (Olanetsoft) [inline data/terms.json:604]: This is good ✅

File: data/terms.json [hunk 6]
```diff
@@ -582,5 +582,37 @@
...
+  },
+  {
+    "term": "orphan block",
+    "meaning": "It's a block that has been solved within the blockchain network but was not accepted by the network yet. The network uses both blocks until one chain has more verified blocks than the other. Then, the blocks in the shorter chain are orphaned."
```
Comment (Olanetsoft) [inline data/terms.json:608]: Some minor update here.

Kindly make it `Orphan block is ...` not `It's a block...`

File: data/terms.json [hunk 7]
```diff
@@ -582,5 +582,37 @@
...
+  },
+  {
+    "term": "orphan block",
```
Comment (Olanetsoft) [inline data/terms.json:611]: You repeated this twice

**Review by Olanetsoft — CHANGES_REQUESTED** [review]

File: data/terms.json (hunk 7 above)
Comment (FrancescoXX) [reply data/terms.json:611]: done

File: data/terms.json (hunk 6 above)
Comment (FrancescoXX) [reply data/terms.json:608]: done

File: data/terms.json (hunk 4 above)
Comment (FrancescoXX) [reply data/terms.json:600]: done

File: data/terms.json (hunk 3 above)
Comment (FrancescoXX) [reply data/terms.json:596]: done

**Review by Olanetsoft — APPROVED** [review]

**Pull request merged**